src/utils.ts

```diff
@@ -16,9 +16,11 @@
  */
 
 import { normalize, sep } from 'node:path';
+// eslint-disable-next-line import/named
+import { TaskMeta } from 'vitest';
```

Comment:
Can we import it as `import * as vitest from 'vitest';` to avoid skipping linter checks?